ocl/load-concepts-to-ocl.sh

```diff
@@ -17,7 +17,8 @@ MYSQL_DOCKER_CONTAINER_NAME=mysql-oclexport
 MYSQL_DOCKER_CONTAINER_PORT=3309
 SDK_TOMCAT_PORT=8080
 SDK_DEBUG_PORT=5000
-OCL_API_URL=https://api.staging.openconceptlab.org
+# OCL_API_URL=https://api.staging.openconceptlab.org
+OCL_API_URL=https://api.openconceptlab.org
```

Comment:
I probably wouldn't commit this change, just to ensure no one accidentally runs this and wipes out all of our sources in production ocl